src/backend/TrafficCourts/Citizen.Service/Models/Disputes/DisputantContactInformation.cs

```diff
@@ -14,35 +14,35 @@ public class DisputantContactInformation
     /// </summary>
     [JsonPropertyName("email_address")]
     [MaxLength(100)]
-    public string? EmailAddress { get; set; } = null!;
+    public string? EmailAddress { get; set; }
```

Comment:
So you _don't_ want to initialize these to null?

Reply:
No because it's not needed. These properties are already defined as optional/nullable so if no value is provided it would still be null.